tools/go-cli/build.gradle

```diff
@@ -0,0 +1,75 @@
+ext.dockerImageName = "whisk/go-cli-test"
+ext.dockerContainerName = "go-cli-test"
+apply from: "../../docker.gradle"
+
+ant.condition(property: "family", value: "linux") {
+    os(family: "unix")
+}
+
+ant.condition(property: "family", value: "mac") {
+    os(family: "mac")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "amd64")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "x86_64")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "i386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "x86_32")
+}
```

Comment:
If we want to defer that change: please add a comment above the ant calls describing that they are used to detect the environment you're running on and set `ant.property.arch` and `ant.property.family` to the respective values of your system